README.md

```diff
@@ -56,12 +56,18 @@ Raspberry Pi 2/3/4 with GNU/Linux.
 ## Prerequisites
 * Code generator in AMaCC relies on several GNU/Linux behaviors, and it
   is necessary to have Arm/Linux installed in your build environment.
+* Throughout this document, Ubuntu - Debian based Linux - on x86 PC is used as an
```

Comment:
Explain why Linux system is necessary for self-hosting and validation.

Reply:
Some errors at RPi 4B, will check further and update later, if any.